Formula/s/screenpipe.rb

```diff
@@ -15,6 +15,7 @@ class Screenpipe < Formula
   depends_on "cmake" => :build
   depends_on "pkgconf" => :build
   depends_on "rust" => :build
+  depends_on "bun"
```

Comment:
we dont have bun formula

Reply:
![Uploading Screenshot 2024-12-09 at 6.39.50 PM.png…]()

Reply:
<img width="1512" alt="Screenshot 2024-12-09 at 6 40 10 PM" src="https://github.com/user-attachments/assets/51edd30b-60ed-405f-825d-da7c65d5a3ba">